.github/workflows/publish.yml

```diff
@@ -0,0 +1,72 @@
+name: Build and publish IMA Agent
+
+on:
+  pull_request:
+    types: [closed]
+    branches:
+      - 'v*.*.*'
+      - develop
+      - beta
+      - stable
+  push:
+    tags:
+      - 'custom-release-*'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Need to add `if: github.event.pull_request.merged == true` to avoid publishing of closed PRs